apis/apim/v1alpha1/ratelimitpolicy_types.go

```diff
@@ -77,6 +77,9 @@ type NetworkingRef struct {
 
 // RateLimitPolicySpec defines the desired state of RateLimitPolicy
 type RateLimitPolicySpec struct {
+	//+listType=map
```

Comment:
nice way to leverage builtin validation support of unique list (not scalar) elements

Reply:
Where did you find doc to do that? I can only find https://book.kubebuilder.io/reference/markers/crd-processing.html
But it does not clearly state it as a validation tool, but a processing behavior configuration.

Reply:
I used the one you mentioned. Also, saw similar usage somewhere in the Kubernetes source code and thought it worth our use case.